### ISSUE TITLE
Initial installation fails with 'Cannot read property 'theme' of undefined'

### ISSUE THREAD

**Issue by FabianMk** [jacekschae/shadow-cljs-tailwindcss#67]
Steps to reproduce (on macOS):
```
npm install 
npm run postcss:build
```
Result:
```
Processing src/css/tailwind.css...

TypeError: Cannot read property 'theme' of undefined
    at _default ([...]/shadow-cljs-tailwindcss/node_modules/tailwindcss/lib/lib/substituteScreenAtRules.js:16:5)
```
How can I fix it?

**Comment by evedes** [comment]
I've PRed a fix for this here: #69

**Comment by jacobemcken** [comment]
I experienced the same error: `TypeError: Cannot read property 'theme' of undefined`
The PR above also solved the issue for me.

**Comment by jacekschae** [comment]
Closed with #69
